com.rustdesk.RustDesk.metainfo.xml

```diff
@@ -1,12 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>com.rustdesk.RustDesk</id>
-  <developer_name>RustDesk</developer_name>
+  <developer id="com.rustdesk">
+    <name>RustDesk</name>
+  </developer>
   <launchable type="desktop-id">com.rustdesk.RustDesk.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>AGPL-3.0-only</project_license>
   <name>RustDesk</name>
-  <summary>An open-source remote desktop application designed for self-hosting, as an alternative to TeamViewer.</summary>
+  <summary>Secure remote desktop access</summary>
```

Comment:
Where did you get this text? How about
```suggestion
  <summary>Full-featured remote desktop app</summary>
```

At least I see this line on the Rustdesk's website here:

![image](https://github.com/user-attachments/assets/bfe54c8c-a525-4784-b51a-5efaa1b9876a)

Reply:
> Where did you get this text? 

The footer of the RustDesk website:

![Skjermbilde fra 2024-10-09 12-56-01](https://github.com/user-attachments/assets/3424b380-2390-4cff-9d1f-3cba593f6da4)

I have no strong opinion on it, but Flathub's quality guidelines recommends avoiding generic nouns like "app" or "tool" in the summary.